app/experimenter/nimbus-ui/src/components/PageSummary/index.test.tsx

```diff
@@ -438,6 +408,54 @@ describe("PageSummary", () => {
     );
   });
 
+  it("handles approval of live update as expected", async () => {
```

Comment:
You could've split this part out.

1) PR for saving LIVE(IDLE/DIRTY) rollout saves and updates fields and lands in LIVE(DIRTY)
2) PR for LIVE(DIRTY) can go through the review/update flow

I know there was already a PR for the review/update flow.  It's not a big deal to include it in here but that could be a way to isolate PR concerns.

Reply:
That's fair. The other PR was pretty much just adding the UI, and I saw this one as actually adding the logic 🤷‍♀️